custom/up_nrhm/data_sources/asha_facilitators.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
You don't need to use `load_spec` for these. They are auto-loaded via a setting: https://github.com/dimagi/commcare-hq/pull/5535/files#diff-bdf3ecebd8379ca98cc89e545fc90899R1238

Here is the class that loads them: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/models.py#L267-267

Steps to test locally:
* Run CommCare HQ and enable the 'User configurable reports UI' feature flag (http://localhost:8000/hq/flags/edit/user_reports/)
* Navigate to: http://localhost:8000/a/up-nrhm/configurable_reports/
* Select the datasource ("ASHA Facilitators")
* Click on the "Rebuild" button at the bottom of the form (celery must be running)

If you want to enable continuous updates to the data you need to run the `CustomDataSourcePillow`:

    ./manage.py run_ptop --pillow-name=CustomDataSourcePillow